turbo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**", "build/**"],
+      "dependsOn": ["^build"]
+    },
+    "lint": {}
```

Comment:
do we still have lerna.json and the deps? if so mind removing?

Reply:
Nah, lerna is already gone